extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/util/RuleEliminateMarker.scala

```diff
@@ -22,6 +22,9 @@ import org.apache.spark.sql.catalyst.rules.Rule
 
 class RuleEliminateMarker extends Rule[LogicalPlan] {
   override def apply(plan: LogicalPlan): LogicalPlan = {
-    plan.transformUp { case rf: RowFilterAndDataMaskingMarker => rf.child }
+    plan.transformUp {
+      case rf: DataMaskingMarker => rf.child
+      case rf: RowFilterMarker => rf.child
```

Comment:
```suggestion
      case DataMaskingMarker(child, _) => child
      case RowFilterMarker(child) => child
```

Reply:
duplicated `rf` is more like the short form of `rowfilter` and not suitable for both DataMaskingMarker and RowFilterMarker. And in suggested style we could just skip naming it .

Reply:
The suggested style is not a good habit. it makes refactoring more difficult and the code more error-prone.

Reply:
```
      case dm: DataMaskingMarker => dm.child
      case rf: RowFilterMarker => rf.child
```
OK, back to the existing style. How about this?